.github/workflows/ci.yml

```
@@ -0,0 +1,37 @@
+name: CI
+on:
+  push:
+  pull_request:
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
```

```suggestion
        python-version: ['3.7', '3.8']
```
This will be good habit because these break once you start testing python 3.10, because YAML interprets these as floats, and when it gets stringified it will be `'3.1'` (ask me how I know 🙃)